src/gui/sdlmain.cpp

```diff
@@ -1857,10 +1866,23 @@ Bitu GFX_SetSize(int width,
 		assert(emptytex);
 
 		memset(emptytex, 0, texture_area_bytes);
-		glTexImage2D(GL_TEXTURE_2D, 0, GL_RGBA8, texsize_w, texsize_h,
+
+		int is_framebuffer_srgb_capable;
+		SDL_GL_GetAttribute(SDL_GL_FRAMEBUFFER_SRGB_CAPABLE,
+		                    &is_framebuffer_srgb_capable);
+
+		sdl.opengl.srgb_framebuffer = is_sharp_shader() && is_framebuffer_srgb_capable > 0;
```

Comment:
For the name `srgb_framebuffer`, when read on its own, there's nothing to tell me that it's a boolean; it could be the actual framebuffer address for all I know. 

Suggest something like: `framebuffer_is_srgb` (or something like that).

Reply:
Fair point, first I called it `use_framebuffer_is_srgb` but didn't quite like it. `framebuffer_is_srgb_encoded` is better, changing to that.